rs/nns/governance/src/neuron_store/metrics.rs

```diff
@@ -98,6 +100,33 @@ impl NeuronMetrics {
         self.public_neuron_subset_metrics
             .increment(voting_power_economics, now_seconds, neuron);
     }
+
+    /// This could modify either still_losing_voting_power_neuron_subset_metrics, or
+    /// done_losing_voting_power_neuron_subset_metrics (but not both), since
+    /// they are very closely related. In particular,
+    fn increment_still_losing_voting_power_or_done_losing_voting_power_neuron_subset_metrics(
+        &mut self,
+        voting_power_economics: &VotingPowerEconomics,
+        now_seconds: u64,
+        neuron: &Neuron,
+    ) {
+        let seconds_since_voting_power_refreshed =
+            now_seconds - neuron.voting_power_refreshed_timestamp_seconds();
+        let Some(seconds_losing_voting_power) = seconds_since_voting_power_refreshed
+            .checked_sub(voting_power_economics.get_start_reducing_voting_power_after_seconds())
+        else {
+            return;
+        };
+
+        if seconds_losing_voting_power < voting_power_economics.get_clear_following_after_seconds()
+        {
+            self.still_losing_voting_power_neuron_subset_metrics
+                .increment(voting_power_economics, now_seconds, neuron);
+        } else {
+            self.done_losing_voting_power_neuron_subset_metrics
+                .increment(voting_power_economics, now_seconds, neuron);
+        }
+    }
```

Comment:
This is where the new stats are compiled. Basically, we just call an existing increment method on the object corresponding to the applicable subcategory.